vue_firebase_option/src/views/Dashboard.vue

```diff
@@ -55,21 +74,47 @@ export default {
     .catch(() => {
       this.alertMessage = 'Cannot get balance. Please reload!'
       this.isError = true
-    });
+    })
+
+    db.collection("users").get().then((querySnapshot) => {
```

Comment:
シングルクォーテーションを使いましょう。
```suggestion
    db.collection('users').get().then((querySnapshot) => {
```